csc_335_restaurant/src/model/Server.java

```diff
@@ -8,56 +8,47 @@
 package model;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 
 public class Server {
-	// private ArrayList<Table> tables;
+	private ArrayList<Integer> tables;
 	private String name;
-	private int num_seated;
 	private double tips;
 	
 	public Server(String name) {
-		//tables = new ArrayList<Table>();
+		this.tables = new ArrayList<Integer>();
 		this.name = name;
-		num_seated = 0;
-		tips = 0.0;
+		this.tips = 0.0;
 	}
 	
-	/*
-	public void seatTable(Tabel t) {
-		// when every table is sat num_seated is updated to add the num of seats from that table
-		tables.add(t);
-		num_seated += t.getSeats();
+	public void addTable(int table) {
+		tables.add(table);
```

Comment:
Should check for duplicate before adding
```suggestion
		if (!tables.contains(table)) {
		    tables.add(table);
		}
```